src/ext/java/org/opentripplanner/ext/siri/updater/azure/SiriAzureETUpdater.java

```diff
@@ -106,13 +107,15 @@ private void processMessage(String message, String id) {
       }
 
       super.saveResultOnGraph.execute((graph, transitModel) -> {
-        snapshotSource.applyEstimatedTimetable(
+        var result = snapshotSource.applyEstimatedTimetable(
           fuzzyTripMatcher(),
           entityResolver(),
           feedId,
           false,
           updates
         );
+        ResultLogger.logUpdateResult(feedId, "siri-et", result);
```

Comment:
This is potentially called several times per second for each incoming update. Is that what you want?